interpreter/cling/lib/Interpreter/BackendPasses.cpp

```diff
@@ -366,9 +373,28 @@ void BackendPasses::CreatePasses(llvm::Module& M, int OptLevel,
   // higher optimization levels we run the normal inliner.
   if (OptLevel <= 1) {
     MPM.addPass(AlwaysInlinerPass());
+
+    // Register a callback for disabling all other inliner passes.
+    PIC.registerShouldRunOptionalPassCallback([](StringRef P, Any) {
+      if (P.contains("InlinerPass"))
+        return false;
```

Comment:
Very likely, this condition is too broad matching all `InlinerPass`es. But please confirm locally that future changes work on platforms where they didn't before, we should not waste CI resources for this...